articles/140_topic_and_service_name_mapping.md

```diff
@@ -305,25 +303,25 @@ It may be possible, through an option in the API or through some name syntax, to
 
 For example with an option in the API:
 
-| ROS Name              | ROS Prefix | DDS Topic   | DDS Partition               |
-|-----------------------|------------|-------------|-----------------------------|
-| `rostopic://image`    | with       | `image`     | [`rt`]                      |
-| `rostopic://image`    | without    | `image`     | [``]                        |
+| ROS Name              | ROS Prefix | DDS Topic   | DDS Partition                 |
+|-----------------------|------------|-------------|-------------------------------|
+| `rostopic://image`    | with       | `image`     | \[`rt`\]                      |
+| `rostopic://image`    | without    | `image`     | \[\]                          |
```

Comment:
This change depends on the feedback to this question: https://github.com/ros2/design/pull/109/files#r125335736